pkg/konfluxgen/konfluxgen.go

```diff
@@ -416,8 +416,17 @@ func Sanitize(input interface{}) string {
 }
 
 func Truncate(input interface{}) string {
+	const (
+		longest = 63
+	)
+
 	in := fmt.Sprintf("%s", input)
-	// TODO very basic name sanitizer
+	if len(in) <= longest {
+		return in
+	}
+
+	in = strings.ReplaceAll(in, "release-v", "")
+	in = strings.ReplaceAll(in, "release-", "")
```

Comment:
maybe we remove this for all (not only, when it's too long (for consistency)) :shrug:

Reply:
yeah, that forces to delete all existing resources, while we need to delete only the ones with too long names

Reply:
should I delete it everywhere? we would need to delete all components / applications

Reply:
ImageRepositories probably too. As it's only eventing components ATM and we have the autoapply, I'd be fine with that and would prefer consistency.